src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Collect.cs

```diff
@@ -19,7 +19,8 @@
 using Microsoft.PowerFx.Syntax;
 using Microsoft.PowerFx.Types;
 using CallNode = Microsoft.PowerFx.Syntax.CallNode;
-using RecordNode = Microsoft.PowerFx.Core.IR.Nodes.RecordNode;
+using IRCallNode = Microsoft.PowerFx.Core.IR.Nodes.CallNode;
```

Comment:
>IRCallNode [](http://example.com/codeflow?start=6&length=10)

I like the IR prefix to avoid collisions. we should follow that convention more often.